tests/unit/h/views/api/orcid_test.py

```diff
@@ -203,9 +195,6 @@ def user(self, factories):
     @pytest.fixture
     def pyramid_request(self, pyramid_request, user):
         pyramid_request.user = user
-        state = "test_state"
-        pyramid_request.params = {"code": "test_code", "state": state}
-        pyramid_request.session["oauth2_state"] = state
```

Comment:
Very hidden, but here a fixture in the view unittests encodes some knowledge of internal details of the schema: the `"oauth2_state"` key string that the schema uses to stash a copy of the OAuth 2 `state` param in the Pyramid session. This is entirely the schema's private business, nothing to do with the views. When trying to change this key string the view tests started failing.

The view tests also don't need to set `pyramid_request.params` anymore: the views don't read the request params, the schema does, and the views just read the validated data that the schema returns.